src/Compilers/CSharp/Portable/Lowering/Instrumentation/DynamicAnalysisInjector.cs

```diff
@@ -71,6 +85,43 @@ private DynamicAnalysisInjector(MethodSymbol method, BoundStatement methodBody,
             }
         }
 
+        private static bool IsExcludedFromCodeCoverage(MethodSymbol method)
+        {
+            if (method.ContainingType is SourceNamedTypeSymbol sourceNamedType &&
```

Comment:
I believe we have helpers to check against well known attributes.

---
In reply to: [91632361](https://github.com/dotnet/roslyn/pull/15720#discussion_r91632361) [](ancestors = 91632361)